servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultSingleAddressHttpClientBuilder.java

```diff
@@ -179,9 +179,9 @@ private DefaultSingleAddressHttpClientBuilder(@Nullable final U address,
                 globalDnsServiceDiscoverer());
     }
 
-    static <U> DefaultSingleAddressHttpClientBuilder<U, InetSocketAddress> forResolvedAddress(
-            final U u, final InetSocketAddress address) {
-        ServiceDiscoverer<U, InetSocketAddress, ServiceDiscovererEvent<InetSocketAddress>> sd =
+    static <U, R> DefaultSingleAddressHttpClientBuilder<U, R> forResolvedAddress(
```

Comment:
Should `R` extend `SocketAddress`?

Reply:
done. somewhat on the fence if we need to take a dependency on the marker base class in our APIs, but we should be able to relax the generic constraints later if necessary.